src/Generating/Trees.cpp

```diff
@@ -301,26 +307,34 @@ void GetTreeImageByBiome(Vector3i a_BlockPos, cNoise & a_Noise, int a_Seq, EMCSB
 			return;
 		}
 
-		case biColdTaiga:
-		case biColdTaigaHills:
 		case biMegaTaiga:
 		case biMegaTaigaHills:
-		case biExtremeHillsPlus:
-		case biSunflowerPlains:
-		case biDesertM:
-		case biExtremeHillsM:
-		case biFlowerForest:
-		case biTaigaM:
-		case biIcePlainsSpikes:
-		case biJungleM:
-		case biJungleEdgeM:
-		case biColdTaigaM:
+		{
+			// TODO: implement trees
```

Comment:
Please could you specify what type of trees are expected in this comment? Just to act as a reminder when they are eventually added.

Reply:
I have changed it.